README.md

```diff
@@ -18,9 +18,12 @@ Data is transformed from these standard formats and loaded into Elasticsearch (t
     - **-t, --transformer** *Default: transformers/[[data file basename]].js* The transformer to use. This converts state-specific data formats to our [Point Data Schema](https://github.com/cfpb/grasshopper/blob/master/docs/point_data_spec.md)
     - **-h, --host** *Default: localhost* The elasticsearch host
     - **-p, --port** *Default: 9200* The elasticsearch port
+    - **-l, --log** *Default: debug* The elasticsearch log level
     - **--index** *Default: address* The elasticsearch index
     - **--type** *Default: point* The elasticsearch type
     - **--profile** *Default: default* The aws credentials profile in ~/.aws/credentials. AWS keys as environment variables will override this setting.')
+    - **--s_srs** *Auto-detects if possible* Source spatial reference system. Needed for untransformed .csv files.
```

Comment:
Yep, I saw that as soon as I dug down a bit deep.  I guess it depends on who the target audience of this tool is.  If most everyone who'd use this is already familiar with `ogr2ogr` and all its warts, then yes, it does make sense to keep as-is.